docs/subgraph-stack.md

```diff
@@ -0,0 +1,66 @@
+Subgraph-Stack
+===
+Stack structure used to manage subgraph storage and access for recursive graph algorithms. Manages a stack of undirected graphs (the member list) {G_0, ..., G_l}  such that G_i is a subgraph of G_{i-1} for each 0 < i < l+i. Each graph on the stack can be accessed and implements the graph access interface (*head(v,k)*, *mate(v,k)*, *deg(v)*, *order()*). The graphs on the stack are not the actual subgraph, but isomorph to the subgraph. I.e. from the view of the user each subgraph G_i(V_i, E_i) has sets V_i={1,...,n_i} and the equivalent set E_i. The structure provides various translation operations to translate from a graph G_i to some other graph G_j on the stack. 
+
+## Operations
+- *push(B_V,B_A)* pushes a new subgraph G_{l+1} on the stack. V is a bitset of length n_l with bits set to *1* at an index v if G_{l+1} contains the vertex v \in V_l. B_V is a bitset of length 2m_l with bits set to *1* at an index a and a' if G_{l+1} contains the edge represented by the arcs a and a' of G_l.
```

Comment:
Did you mean `B_V` instead of `V` and `B_A` instead of `B_V`?

Reply:
yes, fixed